lib/Myriad/RPC/Message.pm

```diff
@@ -0,0 +1,67 @@
+package Myriad::RPC::Message;
+
+use strict;
+use warnings;
+
+# VERSION
+
+use Object::Pad;
+use Syntax::Keyword::Try;
+use JSON::MaybeUTF8 qw(decode_json_utf8 encode_json_utf8);
+
+use Myriad::Exception::BadMessageEncoding;
+
+class Myriad::RPC::Message;
+
+has $rpc;
+has $id;
+has $who;
+has $deadline;
+
+has $args;
+has $stash;
+has $response;
+has $trace;
+
+method id {$id};
```

Comment:
`method name { ... }` is a full statement and doesn't need a `;` after the closing brace. But see additionally the style comments above about putting more whitespace inside them.